qa-ibd.planx-pla.net/manifest.json

```
@@ -4,25 +4,25 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2020.02",
+    "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "dashboard": "quay.io/cdis/gen3-statics:1.0.0",
-    "fence": "quay.io/cdis/fence:2020.02",
-    "indexd": "quay.io/cdis/indexd:2020.02",
-    "peregrine": "quay.io/cdis/peregrine:2020.02",
-    "pidgin": "quay.io/cdis/pidgin:2020.02",
-    "revproxy": "quay.io/cdis/nginx:1.15.5-ctds",
-    "sheepdog": "quay.io/cdis/sheepdog:2020.02",
-    "portal": "quay.io/cdis/data-portal:2.24.8",
+    "fence": "quay.io/cdis/fence:2020.03",
+    "indexd": "quay.io/cdis/indexd:2020.03",
+    "peregrine": "quay.io/cdis/peregrine:2020.03",
+    "pidgin": "quay.io/cdis/pidgin:2020.03",
+    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
+    "sheepdog": "quay.io/cdis/sheepdog:2020.03",
+    "portal": "quay.io/cdis/data-portal:2020.03",
```

```suggestion
    "portal": "quay.io/cdis/data-portal:2.24.8",
```
the portal fix we need is in 2.24.8, which was released after 2020.03

---

Just a minor note: the March release (2020.03) testing comprises the validation of all 2020.03 docker images together (integration tests), hence, by bumping up the version of one single component, you're introducing a risk (which is mitigated by the PR checks / our CI pipeline).
I guess it's just to bear in mind that, ideally, we should strive to stick with the monthly release bundle (of course, I understand there are customer needs at stake so that's ok. I just wanted to reiterate this here).

---

we know that, but there's a bugfix we need that was released after 2020.03